mod/Jailbreak.Warden/Global/WardenPeaceBehaviour.cs

```diff
@@ -308,6 +297,19 @@ public void UnmutePrevMutedPlayers(MuteReason reason, params CsTeam[] targets)
 
     }
 
+    public void UnmutePrevMutedDeadPlayers()
+    {
+        if (_roundEnd)
```

Comment:
todo I should remove this check from inside the function and have the method that calls this UnmutePrevMutedDeadPlayers() function check for round end instead.